asllib/doc/LocalStorageDeclarations.tex

```diff
@@ -330,7 +330,7 @@ \subsubsection{Prose}
 \begin{itemize}
   \item All of the following apply (\textsc{int}):
   \begin{itemize}
-    \item $\lhs$ is a \wellconstrainedintegertype{} with no constraints;
+    \item $\lhs$ is a \pendingconstrainedintegertype{};
```

Comment:
Perhaps add text to the description of the function above that in case of an unconstrained integer type, the result is a type error.